Algorithm-Finders-Keepers.md

```diff
@@ -52,7 +61,18 @@ findElement([1, 2, 3, 4], function(num){ return num % 2 === 0; });
 
 :rocket: [Run Code](https://repl.it/CLn6/0)
 
-## Another Code Solution:
+### Code Explanation:
+
+- To make the code easier, create an undefined variable that will be returned.
+- Loop through the array to check for each element if it satisfy the function. This is done by passing the arr[index of the loop] as the parameter for the function from the second argument.
+- If true, then store the array element, and return it. This will stop the loop. No else needed.
+- If the loop was not broken and it has ended, then return **num** which by default is undefined. This means that none of the elements from the array satisfied the function.
+
+#### Relevant Links
+
+- [JS For Loops Explained](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-For-Loops-Explained)
```

Comment:
~~[JS For Loops Explained](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-For-Loops-Explained)~~ [JS For Loops Explained](JS-For-Loops-Explained)